### PR TITLE
fixed: constructor has been made explicit

### PR DESCRIPTION
**Pull request by akva2**
no excuse, i forgot to test https://github.com/OPM/opm-output/pull/193 with downstreams

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
shameful self-merge to restore build.

File: opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp [hunk 1]
```diff
@@ -413,12 +413,12 @@ namespace Opm
...
-                                                         {"TEMP" , UnitSystem::measure::temperature},
-                                                         {"RS" , UnitSystem::measure::gas_oil_ratio},
-                                                         {"RV" , UnitSystem::measure::oil_gas_ratio},
+        std::map<std::string, RestartKey> solution_keys {{"PRESSURE" , RestartKey(UnitSystem::measure::pressure)},
+                                                         {"SWAT" , RestartKey(UnitSystem::measure::identity)},
+                                                         {"SGAS" , RestartKey(UnitSystem::measure::identity)},
+                                                         {"TEMP" , RestartKey(UnitSystem::measure::temperature)},
+                                                         {"RS" , RestartKey(UnitSystem::measure::gas_oil_ratio)},
+                                                         {"RV" , RestartKey(UnitSystem::measure::oil_gas_ratio)},
                                                          {"SOMAX", {UnitSystem::measure::identity, false}},
```
Comment (atgeirr) [inline opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp]: Strange that this line compiles with no change. Do you know why this does not need the explicit construction?

Comment (akva2) [inline opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp]: Because it uses a two param constructor.

Comment (atgeirr) [inline opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp]: Of course, thanks!

**Pull request merged**